tools/android/set_up_android.py

```diff
@@ -37,59 +37,61 @@ def read_local_properties():
           sdkDir = line.split('=')[1].strip()
   return sdkDir
 
+def test_path(title, path, subpath):
+  test = os.path.join(path, *subpath)
```

Comment:
We plan to migrate all scripts to python3. Does it make sense to use Path here? Generally, the code can be more readable and take less lines :)

Reply:
Sorry, I didn't get it..
(and I have a small experience with python so far)
How is it related to migration to 3?
Do you suggest passing preconcatenated path+subpath to the function?
Could you be more specific please?

Reply:
https://docs.python.org/3/library/pathlib.html

Reply:
> For low-level path manipulation on strings, you can also use the os.path module. 

Yeap, pathlib is more modern, but [`os.path`](https://docs.python.org/3/library/os.path.html) is not deprecated either.

It'll be good to have a coordinated migration effort.
But I would rather separate migration from this PR's scope, unless you insist on it.